deploy/example.yaml

```diff
@@ -0,0 +1,207 @@
+
+# This configuration is for OpenEBS Installation, ControlPlaneUpgrade
+# and unInstallation.
+apiVersion: dao.mayadata.io/v1alpha1
+kind: OpenEBS
+metadata:
+  name: install-openebs-1.5
+  # Namespace i.e. the namespace where the OpenEBS operator and the
+  # other openebs components will be installed/needs-to-be-installed.
+  #
+  # Defaults to “openebs” namespace.
```

Comment:
No need to mention defaults to. It should default to default if nothing is mentioned.